components/dashboard/src/settings/Plans.tsx

```diff
@@ -131,7 +131,7 @@ export default function () {
         } else if (!pollAccountStatementTimeout) {
             // Refresh account statement in 5 seconds in order to poll for upgrade confirmed
             pollAccountStatementTimeout = setTimeout(async () => {
-                const statement = await server.getAccountStatement({});
+                const statement = await getGitpodService().server.getAccountStatement({});
```

Comment:
Why not re-use `server` here?

Reply:
`server` ist the `json-rpc` proxy interface, which on reconnects of websocket might get obsolete.